README.md

```diff
@@ -96,6 +99,67 @@ mongodb-service-0 # Uses some custom k8s service name. Risks being a duplicate e
 If you run the sidecar alongside such a cluster, it may lead to a broken replica set, so make sure to test it well before
 going to production with it (which applies for all software).
 
+#### MongoDB Command
+The following is an example of how you would update the mongo command enabling ssl and using a certificate obtained from a secret and mounted at /data/ssl/mongodb.pem
+
+Command
+```
+        - name: my-mongo
+          image: mongo
+          command:
+            - mongod
+            - "--replSet"
+            - heroku
+            - "--smallfiles"
+            - "--noprealloc"
+            - "--sslMode"
+            - "requireSSL"
+            - "--sslPEMKeyFile"
+            - "/data/ssl/mongodb.pem"
+            - "--sslAllowConnectionsWithoutCertificates"
```

Comment:
They default to false...

https://github.com/cvallance/mongo-k8s-sidecar/pull/21/files#diff-832005ddd5bb8e459623773a6c5ffd67R101

... which should be the safest...

The example in the readme is provided for the user to use as needed so its not really a default. We could display/convey this in a different way so that people don't get this as a copy/paste default if it is felt to be necessary.

Its important however as none of this will work without these params without a bit of extra work getting valid ssl certs or a ca updated, etc...